docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2870,14 +2870,14 @@ Here is the configuration of the syslog regex parser:
 
     Detector *: StreamScan
     Enabled: checked
-    Alert pipe: /usr/local/pf/var/streamscan (To create the fifo file, do: mkfifo /usr/local/pf/var/streamscan)
+    Alert pipe: /usr/local/pf/var/cds (To create the fifo file, do: mkfifo /usr/local/pf/var/cds)
```

Comment:
Put the creation of the fifo in the rsyslog configuration section to avoid confusion.